pkg/quantile/sparse.go

```diff
@@ -57,6 +57,11 @@ func (s *Sketch) Reset() {
 	s.bins = s.bins[:0] // TODO: just release to a size tiered pool.
 }
 
+// GetRawBins return raw bins information as string
+func (s *Sketch) GetRawBins() (int, string) {
+	return s.count, strings.Replace(s.bins.String(), "\n", "", -1)
```

Comment:
using `strings.ReplaceAll` would be more convenient here right?

Reply:
There's only one newline at the end of the string.